test/controllers/observer_controller_test.rb

```diff
@@ -590,6 +590,11 @@ def test_pattern_search
     params = { search: { pattern: "", type: :observation } }
     get_with_dump(:pattern_search, params)
     assert_redirected_to(controller: :observer, action: :list_observations)
+
+    # Make sure this redirects correctly to list_herbaria not list_herariums.
+    params = { search: { pattern: "", type: :herbarium } }
+    get(:pattern_search, params)
```

Comment:
Looks good!
Thanks for catching this and reducing the number of messages we get.
Suggestion if you feel like it:
```suggestion
    get(:pattern_search, params: params)
```
(See https://docs.rubocop.org/rubocop-rails/2.7/cops_rails.html#railshttppositionalarguments. This is one of the many kinds of offenses that's hidden because we have so many.)